data-prepper-core/src/main/java/org/opensearch/dataprepper/pipeline/server/DataPrepperServer.java

```diff
@@ -67,6 +70,7 @@ private HttpServer createServer() {
 
         createContext(server, listPipelinesHandler, authenticator, "/list");
         createContext(server, shutdownHandler, authenticator, "/shutdown");
+        createContext(server, getTransformedPipelinesBodyHandler, authenticator, "/getPipelineBody");
```

Comment:
Same as above. Should this be `/getPipelineTransformedConfig`. Who knows we may need `/getPipelineConfig` in future to get the config before transformation.